.github/workflows/build.yml

```diff
@@ -153,6 +153,29 @@ jobs:
         run: |
           hack/s3_test/run_test.sh
 
+  test-azblob:
+    runs-on: ubuntu-latest
```

Comment:
We have made some changes in our workflows and we pin runners os now: https://github.com/moby/buildkit/pull/3032. Set `ubuntu-20.04`